rocketpy/rocket/rocket.py

```diff
@@ -348,74 +384,63 @@ def evaluate_center_of_mass(self):
             See :doc:`Positions and Coordinate Systems </user/positions>`
             for more information.
         """
-        # Compute center of mass position
         self.center_of_mass = (
             self.center_of_mass_without_motor * self.mass
             + self.motor_center_of_mass_position * self.motor.total_mass
         ) / self.total_mass
         self.center_of_mass.set_inputs("Time (s)")
         self.center_of_mass.set_outputs("Center of Mass Position (m)")
-
+        self.center_of_mass.set_title(
+            "Center of Mass Position (includes motor and propellant)"
+        )
```

Comment:
```suggestion
        self.center_of_mass.set_title(
            "Center of Mass Position (Rocket + Motor + Propellant)"
        )
```

This should a look better in the plots and is more standardized to what you did in `total_mass`